contrib/examples/walkthrough/ups-instance.yaml

```diff
@@ -7,6 +7,7 @@ spec:
   serviceClassName: user-provided-service
   planName: default
   parameters:
-    credentials:
-      name: root
-      password: letmein
+    inline:
+      credentials:
+        name: root
+        password: letmein
```

Comment:
Also, any reason we don't allow the creds to be stored in a configMap?  Guess that could be done post-beta as long as we provide the right extension point.